index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" href="styles/styles.css">
+</head>
+
+<body>
+<header>
```

Comment:
Over het algemeen ziet je navigatie er goed uit. 

Maar het is beter om klassen toe te kennen aan je nav-element. Want wanneer je bijvoorbeeld onderin de pagina ook nog een keer een navigatie hebt en je gebruikt type selectors in je CSS om styling toe te kennen aan de navigatie, zal deze voor beide van toepassing zijn. 

Dus mijn advies is om eigenlijk altijd class-namen te gebruiken. 

Daarnaast had je jumplinks mogen toevoegen aan de list items om direct naar die sectie toe te kunnen navigeren.